### ISSUE TITLE
Shortcut for committing

### ISSUE THREAD

**Issue by hmeine** [rowanj/gitx#219]
I just installed this fork of GitX (used a simpler one in the past).  One simple thing I am missing now, though, is a keyboard shortcut for committing.  Previously, I could stage, enter my commit message, and then press **Shift-Cmd-Enter** instead of using the touchpad to press the "Commit" button.  (Then, I could use Cmd-Q to return to my terminal and continue my work, not using the pointer for anything but staging files/hunks in the beginning).

**Comment by Uncommon** [comment]
I believe cmd-return works.

**Comment by rowanj** [comment]
As @Uncommon says, this fork of GitX uses ⌘⏎ not ⇧⌘⏎  -  drop the `shift` and it should work as expected.

**Comment by hmeine** [comment]
Indeed, cool.  Maybe it would make sense to support both, in order to accomodate users who have used another fork before?

**Comment by elia** [comment]
I'd prefer a commit menu with exposed shortcuts to help discoverability, that would be a good place to add <kbd>⌥⌘A</kbd> for **Amend** and <kbd>⌘S</kbd>/<kbd>⌘U</kbd> for **Stage** / **Unstage**.
